src/renderer/viz/expressions/transition.js

```diff
@@ -28,23 +28,32 @@ export default class Transition extends BaseExpression {
         this.bTime = this.aTime + Number(duration);
         this.type = 'number';
     }
+
     eval () {
         const time = Date.now();
         this.mix = (time - this.aTime) / (this.bTime - this.aTime);
         return Math.min(this.mix, 1.0);
     }
+
     isAnimated () {
         return !this.mix || this.mix <= 1.0;
     }
+
+    isPlaying () {
+        return this.isAnimated();
```

Comment:
While I understand the difference between isAnimated and isPlaying, I think exposing both methods at the API level might be confusing now. Most of the time the end-user will be interested only in the isPlaying one. Thoughts?

Furthermore, seeing the implementation, what's the difference between isPlaying and isAnimated in most of the expressions? Because it's very subtle, isn't it?

Reply:
As a general pattern I would interpret `isAnimated` as "it has an animation", whether it is active now or not, while `isPlaying` is more expressive: "a paused layer, with an animation inside, is not playing, but it is animated (for example, it might have some special controls in the app)".

In this particular case, it is true I had some doubts, cause `isPlaying` is much more clear in an animation context (where `isAnimated` keeps being always true; see xxx), so I'm open to suggestions.

Reply:
Fair enough. Your reasoning about having one for determining if you should have the controls proves that having both is good.

Let's :shipit:!

Reply:
I think the difference is clear... but slightly confusing when explained: isAnimated means is not a static visualization/expression, but that contains animations and can be *played* (or be stopped, etc.). IsPlaying is an *is-animated* expression that is not stopped.

Reply:
And, yes, I think users should only be interested in `isPlaying`, so `isAnimated` could be renamed `_isAnimated` and kept private.